src/rard/templates/base.html

```diff
@@ -28,6 +28,7 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css">
     <link href="https://quill-mention.github.io/quill-mention/quill.mention.min.css" rel="stylesheet">
     <script src="https://unpkg.com/htmx.org@1.9.2" integrity="sha384-L6OqL9pRWyyFU3+/bjdSri+iIphTN/bvYyM37tICVyOJkWZLpP2vGn6VUEXgzg6h" crossorigin="anonymous"></script>
+    <script src="https://unpkg.com/hyperscript.org@0.9.8"></script>
```

Comment:
What's this (for)?

Reply:
Part of my allergy to javascript. See bibliographyitem_overview.html for an example where it allows embedding show/hide toggle as html tag attribute.